src/02-common-principles.md

```diff
@@ -520,6 +520,18 @@ for more information.
 }
 ```
 
+## Uniform Resource Indicator 
+
+A Uniform Resource Indicator (URI) is a string referring to a resource and SHOULD
+have the form `<scheme>:[//<authority>]<path>[?<query>][#<fragment>]`, as specified 
+in [RFC 3986](https://tools.ietf.org/html/rfc3986). 
+This applies to URLs and other common URIs, including Digital Object Identifiers (DOIs), 
+which may be fully specified as `doi:<path>`, 
+for example, [doi:10.5281/zenodo.3686061](https://doi.org/10.5281/zenodo.3686061).
+A given resource may have multiple URIs.
+When selecting URIs to add to dataset metadata, it is important to consider 
+specificity and persistence.
+
```

Comment:
Does it have to cover URLs specifying the DOI resolver? 

Still trying to wrap my around this: I thought that URLs would be a valid URI anyway.

Reply:
`doi:<thing>` and `https://doi.org/<thing>` are equivalent from an identifier perspective. this may be more of a preference as saying  `doi:<thing>` is preferred. bids may decide to not allow the full url formulation. but that seems a bit more draconian.

Reply:
The question is "What do we mean when we say DOI". I would argue that we mean "doi:<DOI>", not an arbitrary URI that may or may not refer to the same object. Currently all the examples show `<DOI>`, but it seemed like a good idea to include `https://doi.org/<DOI>` as a thing that will be warned (which is all DEPRECATED means until BIDS 2.0) about by the validator.

If that feels like it's overstepping, then I'm okay with changing to: 

```Markdown
Bare DOIs such as `10.18112/openneuro.ds000001.v1.0.0` are DEPRECATED.
```

Reply:
I would say let's make go "small steps" and just have :

```Bare DOIs such as `10.18112/openneuro.ds000001.v1.0.0` are DEPRECATED.```